### PR TITLE
[Frontend] Story pages markup

### PR DESCRIPTION
**Pull request by kakimaru**
# Changes
- [x] Styled stories page
- [x] Create story detail page route
- [x] Styled story detail page

### CODE REVIEW

File: app/(main)/stories/_mockdata.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { Story } from "@/app/types/story";
+
+export const mockData: Story[] = [
+  {
+    id: "1",
+    title: "Title",
```
Comment (benono) [inline app/(main)/stories/_mockdata.ts:6]: @kakimaru 
Good work on the mock data! Here are some suggestions for improvement:

Current Status:
- ✅ Basic structure is well-defined
- ✅ Data types are consistent
- ⚠️ Data entries 1-4 are too similar (same titles, images, and author info)

Suggestions for Improvement:
1. Data Variety
- Add more diverse content:
  - Different title lengths
  - Various author names
  - Different image URLs
  - Realistic dates
  - Unique artwork names

2. Edge Cases
Consider adding entries that test:
- Very long titles
- Special characters in text
- Various author name formats

Overall, the current mock data is well-structured and functional. Adding these edge cases would help ensure the UI remains robust across different scenarios.

**Comment by benono** [comment]
LGTM !!
